menu.js

```diff
@@ -0,0 +1,32 @@
+function openMenu() {
+  const menu = document.querySelector('nav>ul');
+  const menuItem = Array.from(document.querySelectorAll('nav > ul li'));
+  for (let item = 0; item < menuItem.length; item += 1) {
+    menuItem[item].classList.add('menu-item');
+  }
+  menu.classList.add('mobile-menu');
+  document.querySelector('nav > img').classList.toggle('visible');
+  document.querySelector('nav > div').classList.toggle('visible');
+}
+
+function closeMenu() {
+  const menu = document.querySelector('nav>ul');
+  const menuItem = Array.from(document.querySelectorAll('nav > ul li'));
```

Comment:
- [ ] Kindly change these regular functions to `arrow functions`. In the project requirements particularly [this section](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.), it encourages the use of ES6. Javascript's ES6 brings new syntax and awesome features to make your code more modern and readable. It allows you to write less code and do more. ES6 introduces us to many great features like arrow functions. The arrow function is really awesome and makes your code more readable, more structured, and look like modern code. Take a minute and also go through [this article on why ES6 arrow functions are so awesome](https://zendev.com/2018/10/01/javascript-arrow-functions-how-why-when.html).